submit_and_compare/submit_and_compare.py

```diff
@@ -238,26 +248,31 @@ def student_submit(self, submissions, suffix=''):
 
     @XBlock.json_handler
     def studio_submit(self, submissions, suffix=''):
-        '''
+        # pylint: disable=unused-argument
+        """
         Save studio edits
-        '''
+        """
         self.display_name = submissions['display_name']
-        self.weight = self._get_natural_number(submissions['weight'])
-        max_attempts = self._get_natural_number(submissions['max_attempts'])
-        if max_attempts > 0:
+        self.weight = SubmitAndCompareXBlock._convert_to_int(
+            submissions['weight']
+        )
+        max_attempts = SubmitAndCompareXBlock._convert_to_int(
```

Comment:
While we could treat all of these as class methods, there are some benefits to instead defining them as module functions.

One benefit is that we can selectively import the functions we need.
Also, we don't need to worry about distinguishing between `self` and `cls` and/or fully specifying the class name (here: `SubmitAndCompareXBlock`).

Together, this allows us to right less verbose code.